test/service/test_response.py

```diff
@@ -55,7 +54,8 @@ def get_hits(self, entity_type: str, entity_id: str):
         }
         # Tests are assumed to only ever run with the azul dev index
         results = self.es_client.search(index=config.es_index_name(entity_type, aggregate=True), body=body)
-        return Document.translate_fields([results['hits']['hits'][0]['_source']], forward=False)
+        from azul.plugin import Plugin
```

Comment:
If this is to avoid a circular import, I would like to talk about avoiding it. Otherwise, it should be top-level import.

Reply:
Should be top-level import. Will fix.